code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -829,25 +835,15 @@
 
 	if(is_organic_limb())
 		if(should_draw_greyscale)
-//			limb.icon = 'icons/mob/human_parts_greyscale.dmi'
-			switch(species_id) // [FULP EDIT STARTS]
-				if("beefman") //If we ever add more unique races - this should be a list of them.
-					limb.icon = 'fulp_modules/beefman_port/icons/mob/beefman_bodyparts.dmi'
-				else
-					limb.icon = 'icons/mob/human_parts_greyscale.dmi'// [FULP EDIT END]
+			limb.icon = 'icons/mob/human_parts_greyscale.dmi'
```

Comment:
Why did you remove that? Did you implement it differently?

Reply:
Overwritten in ZFulpcode, _bodyparts.dm